packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -303,9 +303,15 @@ export class Nav extends BaseComponent<INavProps, INavState> implements INav {
   }
 
   private _onLinkExpandClicked(link: INavLink, ev: React.MouseEvent<HTMLElement>): void {
+    const { onLinkClick } = this.props;
+
     link.isExpanded = !link.isExpanded;
     this.setState({ isLinkExpandStateChanged: true });
 
+    if (onLinkClick) {
+      onLinkClick(ev, item);
```

Comment:
shouldn't this be link instead of item?

Reply:
also please run `npm run change` to create a change file, and commit it with this PR.